packages/accounts-app/src/Accounts/AccountOverviewDetailed.tsx

```diff
@@ -92,10 +92,10 @@ export function AccountOverviewDetailed (props: Props) {
       .getOrElse(false);
 
     const isStashValidating = fromNullable(allStashes)
-      .map(allStashes => allStashes.includes(new AccountId(currentAccount)))
+      .map(allStashes => allStashes.includes(createType('AccountId', currentAccount)))
```

Comment:
If `allStatches` is a `Vec`, you don't need to convert - you can feed it any representation (see the AccountId comment), and it will use the codec eq to determine if it is equal or not.

Reply:
`allStashes` here is actually a `AccountId[]` so the conversion is still needed but i'll keep that in mind if it comes up elsewhere.